src/fontra/views/editor/editor.html

```diff
@@ -315,7 +317,12 @@
     .sidebar-resize-gutter {
       height: 100%;
       width: 8px;
-      cursor: col-resize;
+      cursor: var(--resize-cursor);
+      position: absolute;
+    }
+
+    .sidebar-container.left .sidebar-resize-gutter {
+      right: 0;
```

Comment:
Suggest to make this `-4px`, and add one for `.right.` as well: `left: -4px`.

This would center the gutter around the edge, instead of being completely inside the tab. I feel this is more natural beahvior.

I actually wonder if 8px width for the gutter isn't too generous. Perhaps 6 is enough?

Reply:
Or even 4px. It seems enough to grab it.